### PR TITLE
Add detailed info when throw RejectedRecordsException.

### PR DESCRIPTION
**Pull request by NeoZHL**
*Issue #, if available:*

*Description of changes:*
Add detailed rejected records info when throw RejectedRecordsException.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sample_apps/java/src/main/java/com/amazonaws/services/timestream/CrudAndSimpleIngestionExample.java [hunk 1]
```diff
@@ -207,6 +209,13 @@ public void writeRecords() {
         try {
             WriteRecordsResult writeRecordsResult = amazonTimestreamWrite.writeRecords(writeRecordsRequest);
             System.out.println("WriteRecords Status: " + writeRecordsResult.getSdkHttpMetadata().getHttpStatusCode());
+        } catch (RejectedRecordsException e) {
+            System.out.println("RejectedRecords: " + e);
+            for (RejectedRecord rejectedRecord : e.getRejectedRecords()) {
+                System.out.println("Rejected Index " + rejectedRecord.getRecordIndex() + ": "
+                        + rejectedRecord.getReason());
+            }
+            System.out.println("Other records have already been wrote successfully. ");
```
[the remark this answers is not in the document]
Comment (ozanozen) [reply sample_apps/java/src/main/java/com/amazonaws/services/timestream/CrudAndSimpleIngestionExample.java:218]: Now that I checked deeper, I noticed it is for different sample apps. Feel free to ignore or apply this comment.

**Review by ozanozen — COMMENTED** [review]
We should also have changes for go and .net.

**Comment by NeoZHL** [comment]
> We should also have changes for go and .net.

Sorry for missing .net, will add them.
For go, it does print rejected records as following:
Error:
RejectedRecordsException: One or more records have been rejected. See RejectedRecords for details.
{
  RespMetadata: {
    StatusCode: 419,
    RequestID: "GCV5GD3IUIQBVEYQHFJDPG2WUI"
  },
  Message_: "One or more records have been rejected. See RejectedRecords for details.",
  RejectedRecords: [{
      Reason: "A record already exists with the same time, dimensions, and measure name. Updating measure value is not supported.",
      RecordIndex: 2
    }]
}

File: sample_apps/java/src/main/java/com/amazonaws/services/timestream/CrudAndSimpleIngestionExample.java (hunk 1 above)
Comment (zhihaoli) [inline sample_apps/java/src/main/java/com/amazonaws/services/timestream/CrudAndSimpleIngestionExample.java:218]: Can you update "wrote" to "written"? Or maybe simplify the phrase to "Other records were written successfully". Same for other sample apps.

**Pull request closed without merging**